doc/classes/Camera.xml

```diff
@@ -121,6 +121,24 @@
 				Sets the camera projection to perspective mode (see [constant PROJECTION_PERSPECTIVE]), by specifying a [code]fov[/code] (field of view) angle in degrees, and the [code]z_near[/code] and [code]z_far[/code] clip planes in world space units.
 			</description>
 		</method>
+		<method name="set_oblique">
+			<return type="void" />
+			<argument index="0" name="fov" type="float" />
+			<argument index="1" name="oblique_data" type="Dictionary" />
+			<argument index="2" name="z_near" type="float" />
+			<argument index="3" name="z_far" type="float" />
+			<description>
+				Sets the camera projection to perspective with oblique near clipping (see [constant PROJECTION_OBLIQUE]), by specifying a [code]fov[/code] (field of view) angle in degrees, the [code]oblique_data[/code] used to transform the projection frustum, and the [code]z_near[/code] and [code]z_far[/code] clip planes in world space before transformation.
+				[b]Note:[/b] The oblique_data consists of {"camera_gt": The camera global transform, "normal": the desired normal vector of the plane, "position": the world-space position of the plane relative to the camera, "offset": an adjustable offset for the oblique plane distance value}
```

Comment:
```suggestion
				[b]Note:[/b] [code]oblique_data[/code] is a dictionary containing the following fields:
				[code]camera_gt[/code]: The camera global transform.
				[code]normal[/code]: the desired normal vector of the plane.
				[code]position[/code]: the world-space position of the plane relative to the camera.
				[code]offset[/code]: an adjustable offset for the oblique plane distance value.
```

Something similar to the `Physics2DDirectSpaceState` documentation.

Please also wrap code (e.g. variable names) inside `[code][/code]` in other descriptions :)

Reply:
Cheers! do I just hit the 'commit suggestion' button to use your changes? and then go back to tidy up the rest?